.chglog/config.yml

```diff
@@ -0,0 +1,27 @@
+style: github
+template: CHANGELOG.tpl.md
+info:
+  title: CHANGELOG
+  repository_url: https://github.com/rancher-sandbox/cOS-toolkit
+options:
+  commits:
+    # filters:
+    #   Type:
+    #     - feat
+    #     - fix
+    #     - perf
+    #     - refactor
+  commit_groups:
+     title_maps:
+       feat: Features
```

Comment:
tests? Maybe we should add those into the ignore?

Reply:
:thinking: I wasn't sure about it either, I've opted for defaults here